Adyen/Service/PosTerminalManagement.cs

```diff
@@ -133,5 +135,29 @@ public async Task<GetTerminalDetailsResponse> GetTerminalDetailsAsync(GetTermina
             var jsonResponse = await _getTerminalDetails.RequestAsync(jsonRequest);
             return JsonConvert.DeserializeObject<GetTerminalDetailsResponse>(jsonResponse);
         }
+        
+        /// <summary>
+        /// post /getStoresUnderAccount
+        /// </summary>
+        /// <param name="getStoresUnderAccountRequest"></param>
+        /// <returns>GetStoresUnderAccountResponse</returns>
+        public GetStoresUnderAccountResponse GetStoresUnderAccount(GetStoresUnderAccountRequest getStoresUnderAccountRequest)
+        {
+            var jsonRequest = Util.JsonOperation.SerializeRequest(getStoresUnderAccountRequest);
+            var jsonResponse =  _getStoresUnderAccount.Request(jsonRequest);
+            return JsonConvert.DeserializeObject<GetStoresUnderAccountResponse>(jsonResponse);
+        }
+        
+        /// <summary>
+        /// post /getStoresUnderAccount
+        /// </summary>
+        /// <param name="getStoresUnderAccountRequest"></param>
+        /// <returns>task GetStoresUnderAccountResponse</returns>
+        public async Task<GetStoresUnderAccountResponse> GetStoresUnderAccountAsync(GetStoresUnderAccountRequest getStoresUnderAccountRequest)
+        {
+            var jsonRequest = Util.JsonOperation.SerializeRequest(getStoresUnderAccountRequest);
```

Comment:
```suggestion
            var jsonRequest = getStoresUnderAccountRequest.ToJson();
```
Might as well use the model serializer.